src/server/game/Miscellaneous/Language.h

```diff
@@ -811,6 +811,9 @@ enum TrinityStrings
     LANG_PINFO_CHR_PLAYEDTIME           = 853,
     LANG_PINFO_CHR_MAILS                = 854,
     LANG_PINFO_CHR_LEVEL_HIGH           = 871,
+    LANG_NO_REASON             			= 35407,
```

Comment:
This too

Reply:
> 
> 
> You guys tested? it would close my issue #1067

Yeah, just tested. Everything is fine, can close #1067  :D
![pinfoposzer](https://user-images.githubusercontent.com/41213210/49345062-fd8dac80-f67f-11e8-985d-670f85a0b59a.png)